docs/en_US/NNICTLDOC.md

```diff
@@ -567,6 +568,54 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl package show
     ```
 
+<a name="feed"></a>
+![](https://placehold.it/15/1589F0/000000?text=+) `nnictl feed`
+
+* Description
+
+  You can use this command to feed several prior or supplementary trial hyperparameters and results for NNI hyperparameter tuning.
```

Comment:
Add: "The data are fed to the tuning algorithm (e.g., tuner or advisor)."

Reply:
Done.